database/migrations/sql/000605_add_stake_tables.up.sql

```diff
@@ -0,0 +1,39 @@
+create table stake
+(
+    wallet_id     bytea     not null,
+    blockchain_id uuid      not null,
+    object_id     uuid      not null,
+    amount        integer   not null,
+    last_comment  integer   not null,
+    updated_at    timestamp not null,
+    created_at    timestamp not null,
+    constraint stake_pk
+        primary key (blockchain_id, wallet_id, object_id),
+    constraint stake_wallet_fk
+        foreign key (blockchain_id, wallet_id) references wallet (blockchain_id, wallet_id)
+            on update cascade on delete cascade,
+    constraint stake_object_fk
+        foreign key (object_id) references object (object_id)
+            on update cascade on delete cascade
+);
+
+create table pending_stake
+(
+    transaction_id bytea       not null,
+    object_id      uuid        not null,
+    wallet_id      bytea       not null,
+    blockchain_id  uuid        not null,
+    amount         numeric(78) not null,
+    comment        text        not null,
+    kind           integer     not null,
+    updated_at     timestamp   not null,
+    created_at     timestamp   not null,
+    constraint pending_stake_pk
+        primary key (blockchain_id, wallet_id, transaction_id),
```

Comment:
I think it is good enougto only have transaction_id be a PK. At least this is how i configured it on dev2, trying to add additional PKs will throw errors on this table as transaction_id in itself is unique enough.